library/src/main/java/com/joom/grip/commons/PathExtensions.kt

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2021 SIA Joom
```

Comment:
Today is 2022 :) And it would be nice to add this header to all .kt files.

Reply:
done

Reply:
and for Readme as well